### DISCUSSION TITLE
Livestock distribution under current and future scenarios

### DISCUSSION THREAD

**Discussion by KeLiu7** [magpiemodel/magpie#445] [Help]
Dear MAgPIE community, 

I get stuck with extracting the current livestock distribution in MAgPIE. 

Could you please guide me on how to show the livestock numbers change and shift to in future? I did not find the relevant outputs when I use `shinyresults`. Ideally, it should have similar results just like the second figure below? 

Looking forward to your suggestions. Many thanks in advance.

![image](https://user-images.githubusercontent.com/72141690/187422472-30674c99-9e98-4576-98bb-3a7fb70e3d97.png)

![1661858103153](https://user-images.githubusercontent.com/72141690/187422846-cdf3ca4c-bf5a-4b85-89f5-42eb9290d8f1.jpg)

**Comment by bodirsky** [comment]
we only have production quantiy, not livestock numbers. Isabelle may know whether its possible to convert production into livestock numbers. If yes, it would be nice if you could make this available for everyone as a function :)

**Reply by KeLiu7** [reply]
> we only have production quantiy, not livestock numbers. Isabelle may know whether its possible to convert production into livestock numbers. If yes, it would be nice if you could make this available for everyone as a function :)
 Thanks! 

@weindl Hi Isabelle, could you please give some hints? thanks in advance

**Reply by weindl** [reply]
Currently, there are no output functions available that return livestock numbers for a given MAgPIE simulation.
